shared/setCookies.ts

```diff
@@ -66,16 +66,7 @@ export function setCookies({
         expirationDate,
         domain,
       })
-      notifyUser({
-        name,
-        url,
-        secure,
-        httpOnly,
-        path,
-        sameSite,
-        expirationDate,
-        domain,
-      })
+      notifyUser(url)
```

Comment:
We are passing this variable also to the setCookie and there we need it with the name url.

Reply:
But wouldn't it make sense to change the name of it in the `setCookie` function as well then? Just to avoid confusion about whether we pass origins or urls 😅

Reply:
There we are passing the cookie to `chrome.cookies.set` and there we needed as a url :D 
<img width="711" alt="Screenshot 2024-08-16 at 10 05 39" src="https://github.com/user-attachments/assets/e1bb1ce2-1b75-449a-b705-8eee1248ee84">

Reply:
I see 😅 All good then